kumascript/tests/macros/jsxref.test.js

```diff
@@ -163,6 +163,7 @@ describeMacro("jsxref", function () {
 
       macro.ctx.info.getPageByURL = jest.fn((url) => {
```

Comment:
I think to make the test correct we have to add
```js
      const default_ref_url = js_ref_url + partial_slug;
      const default_glob_url = `${js_ref_url}Global_Objects/${partial_slug}`;
```
just above and replace the mock with this:
```js
      macro.ctx.info.getPageByURL = jest.fn((url) => {
        url = getPathname(url);
        if (url === glob_url) {
          return {
            url: glob_url,
            slug: js_ref_slug + partial_slug,
          };
        } else if (url === default_glob_url) {
          return {
            url: default_glob_url,
            slug: js_ref_slug + partial_slug,
          };
        } else if (url === default_ref_url || url === ref_url) {
          return {};
        }
      });
```

Reply:
Thanks for suggestions. I've update the testing case and rebased